### ISSUE TITLE
Weapon animations don't match with the sound when you got the skill to reload faster

### ISSUE THREAD

**Issue by ETLegacyTeam** [etlegacy/etlegacy#793]
Author: @saukko28 (Saukko)
Date: 2015-01-20
Redmine Issue: 793, https://dev.etlegacy.com/issues/793

---

As the title says. The reload animation doesn't match with the sound when you got the skill to reload faster. This applies to every weapon.

-*S

---

Related issue(s): #792

---

Related commit(s):
af753b896b2bec660170c5372a1adf5a276b0b89 - mod: fix fast realod anim, refs 793

**Comment by ETLegacyTeam** [comment]
From: @saukko28 (Saukko)
Date: 2015-01-20T22:35:38Z

---

If this would mean that there would need to be done new sound files, I would be able to do that. It would just require to make the reload sound a bit faster, but not too fast and match it with the reload animation. So there'a lot to do.

-*S

**Comment by ETLegacyTeam** [comment]
From: @rmarquis (Spyhawk)
Date: 2015-01-20T22:41:21Z

---

Pitch and frequency would also need to be corrected (not only duration of sound file).

**Comment by ETLegacyTeam** [comment]
From: n00btime (n00btime)
Date: 2015-05-15T07:15:47Z

---

Hi there,
It turns out this is simpler than you think. I "fixed" this bug for etpub back in 2007. ET's reload animations are split into 3 parts (the reasons why escape me at this point). Each is controlled by a weapons.cfg under etmain. The path is etmain/models/weapons2/sten/weapon.cfg. The line which reads (with a comment) RELOAD2, I changed the fps value from 20 to 29. I've regretted for a while that this seems slightly slow =) Perhaps changing it to 30 or 31 would fit nicely. Anyway there's no need to bother with worrying about the sound file's pitch and frequency (id's devs certainly didn't). Hope this helps. (Modified file attached in case I was ambiguous).
